js/scripts.js

```diff
@@ -1,2 +1,50 @@
-let hello = 'Привет';
-console.log(hello);
+const questionOne = {
+  question: 'Где обитает жираф?',
+  answer: {
+    a: 'В Индии',
+    b: 'В Америке',
+    c: 'В Африке',
+    d: 'В Австралии'
+  },
+  currectAnswer: 'В Африке'
+};
+const questionTwo = {
+  question: 'Как быстро может бегать жираф?',
+  answer: {
+    a: '70 км/ч',
+    b: '55 км/ч',
+    c: '30 км/ч',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/voronkova-ab/game-js/pullRequest?prid=7705697)
```suggestion
    c: "30 км/ч",
```